### PR TITLE
Day7 Ex1 Intent

### PR DESCRIPTION
**Pull request by at-ducle**

### CODE REVIEW

File: app/src/main/java/vn/asiantech/internship/MainIntentActivity.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+     */
+    public void switchFragment(Fragment fragment, boolean addToBackStack, int id) {
+        FragmentManager fm = getFragmentManager();
+        FragmentTransaction ft = fm.beginTransaction();
+        if (addToBackStack) {
+            ft.addToBackStack(null);
+        }
+        if (fragment.getTag() == null) {
+            ft.replace(id, fragment, fragment.toString());
+        } else
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/internship/MainIntentActivity.java]: ERROR: 'else' construct must use '{}'s.

File: app/src/main/java/vn/asiantech/internship/fragment/SecondIntentFragment.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                mOnClickButton = (OnClickButton) getActivity();
+                if (mOnClickButton != null) {
+                    mOnClickButton.setClick();
+                }
+            }
+        });
+        return view;
+    }
+
+    public interface OnClickButton {
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/internship/fragment/SecondIntentFragment.java]: ERROR: Missing a Javadoc comment.

File: app/src/main/java/vn/asiantech/internship/MainIntentActivity.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            ft.replace(id, fragment);
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/internship/MainIntentActivity.java]: WARNING: [IfElseStmtsMustUseBraces] Avoid using if...else statements without curly braces
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/braces.html#IfElseStmtsMustUseBraces

**Comment by at-tienhoang** [comment]
LGTM

**Pull request merged**